card.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
sadly will have to get rid of the individual card page 😭 (as discussed) and it becomes story 3